internal/core/drand_beacon_test.go

```diff
@@ -2,6 +2,10 @@ package core
 
 import (
 	"context"
+	"github.com/drand/drand/common/key"
+	"github.com/drand/kyber"
+	"github.com/drand/kyber/share"
+	dkg2 "github.com/drand/kyber/share/dkg"
```

Comment:
There's really something weird with our Golangci linting since it lets this through